__test__/users.test.js

```diff
@@ -56,7 +56,7 @@ describe('PATCH /:username/image/upload', () => {
         expect(res.status).toBe(200);
         expect(res.body.data).toContain(1);
       });
-  });
+  }, 10000);
```

Comment:
can you take out the timeout since you excluding the file???

Reply:
Test is now included in the test and timeout removed